build-tools-internal/version.properties

```diff
@@ -18,8 +18,8 @@ log4j             = 2.11.1
 slf4j             = 1.6.2
 ecsLogging        = 1.2.0
 
-# when updating the JNA version, also update the version in buildSrc/build.gradle
-jna               = 5.7.0-1
+# when updating the JNA version, also update the version in build-tools-internal/build.gradle
```

Comment:
If we leverage `versions` in the reference build script instead we can ditch this comment since updating it here will update the build script as well.

Reply:
:+1: done in e58285b2d15f196e22220f56c9f4cbc4d5189c4f